styles/dataDashboard.css

```diff
@@ -214,6 +214,28 @@ textarea {
     transition: all 0.5s ease;
   } 
 
+  .canvas-outer-bar {
+        position: relative;
+        max-width: 100%;
+        overflow-x: scroll;
+  }
+
+  .canvas-outer-bar .canvas-inner {
+        width: 800px;
+        overflow-x: scroll;
+  }
+
+  .canvas-outer-radar {
+        position: relative;
+        max-width: 100%;
+        overflow-x: scroll;
+  }
+
+  .canvas-outer-radar .canvas-inner {
+        width: 800px;
+        overflow-x: scroll;
+  }
+
```

Comment:
Styles for the new graph wrappers. Feel free to play around with these values. They are canceled out below (in the wider viewport).